insights/parsers/crypto_policies.py

```diff
@@ -0,0 +1,89 @@
+"""
+crypto-policies - files in ``/etc/crypto-policies/back-ends/``
+========================================
+
+This is a collection of parsers that all deal with the generated configuration
+files under the ``/etc/crypto-policies/back-ends/`` folder.  Parsers included
+in this module are:
+
+CryptoPoliciesConfig - file ``/etc/crypto-policies/config``
+----------------------------------------------------
+
+CryptoPoliciesStateCurrent - file ``/etc/crypto-policies/state/current``
+----------------------------------------------------
+
+CryptoPoliciesOpensshserver - file ``/etc/crypto-policies/back-ends/opensshserver.config``
+----------------------------------------------------
+"""
+
+from insights import Parser, parser, SysconfigOptions
+from insights.specs import Specs
+from insights.parsers import SkipException
+
+
+@parser(Specs.crypto_policies_config)
+class CryptoPoliciesConfig(Parser):
+    """
+    This parser reads the ``/etc/crypto-policies/config`` file.  It uses the
+    ``SysconfigOptions`` parser class to convert the file into
+    a dictionary of options. It also provides the ``options`` property as a helper
+    to retrieve the ``CRYPTO_POLICY`` variable.
+
+    Sample Input::
+
+        LEGACY
+
+    Examples:
+        >>> cp_c.value
+        'LEGACY'
+    """
+    def parse_content(self, content):
+        if self.file_path != "/etc/crypto-policies/config":
```

Comment:
Since the `Specs.crypto_policies_config` is defined as `"/etc/crypto-policies/config"`, I think this check condition will never be reached.

How about raising the SkipException for "Empty" content instead?

Maybe you have other concerns, please correct me if I'm wrong.